src/controllers/expenses.js

```diff
@@ -0,0 +1,107 @@
+'use strict';
+
+const expensesService = require('../services/expenses');
+const usersService = require('../services/users');
+
+const getAll = (req, res) => {
+  const searchParams = req.query;
+
+  const expenses = expensesService.getAll(searchParams);
+
+  res.send(expenses);
+};
+
+const findExpenseById = (req, res) => {
```

Comment:
This is expenses controller, so you can write short name
```suggestion
const findById = (req, res) => {
```